scripts/download_hdx_admin_pop.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This script downloads tabular data about population for country code from $1
+# it uses manually created file static_data/hdx_population_tabular.json
+# inside json there are: url to download layer, name for result dataset
+
+country_code=$(basename "${1%%.*}")
+out_dir="data/out/country_extractions/${country_code}/223_popu"
```

Comment:
Is it possible to download to `data/out` ?
mixture of in and out usually is not a good thing.